src/components/menu/styles.ts

```diff
@@ -0,0 +1,68 @@
+import { styled } from '@linaria/react'
+import { Text2XS } from '../typography'
+
+export const ElMenuPopover = styled.div`
+  position: absolute;
+  white-space: nowrap;
+  z-index: 98;
+`
+
+export const ElMenu = styled.div`
```

Comment:
**question:** Shouldn't this be `ElMenuList` instead of `ElMenu`?

Reply:
yes, that is initially because despite this is consumed as `Menu.List` its actually the outer container of the menu card that has styling , but I think its better to match the classname with component name now 👍